src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/ApprovalStatus.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.managecase.client.datastore;
+
+public enum ApprovalStatus {
+
+    NOT_CONSIDERED("0"),
```

Comment:
we are using, 0, 1, 2 as values, instead of creating String code, can we use enum ordinal() method?

Reply:
We could technically, but followed the advice of [here](https://stackoverflow.com/questions/44654291/is-it-good-practice-to-use-ordinal-of-enum) for example to reduce chance of future errors.